pkg/cosign/cosign.go

```diff
@@ -27,50 +25,10 @@ import (
 	sigs "github.com/sigstore/cosign/pkg/signature"
 	"github.com/sigstore/sigstore/pkg/signature"
 	"github.com/sigstore/sigstore/pkg/signature/payload"
-	"k8s.io/client-go/kubernetes"
 )
 
-var (
```

Comment:
Moved this logic from cosign to the registry client package since we need it both for cosign and also the imageRegistry context var.